modules/ROOT/pages/6.0.3-release-notes.adoc

```diff
@@ -0,0 +1,90 @@
+= TinyMCE 6.0.3
+:navtitle: TinyMCE 6.0.3
+:description: Release notes for TinyMCE 6.0.3
+:keywords: releasenotes, new, changes, bugfixes
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+[[overview]]
+== Overview
+
+{productname} 6.0.3 was released for {enterpriseversion} and {cloudname} on Wednesday, June 1^st^, 2022. It includes {productname} 6.0.3. These release notes provide an overview of the changes for {productname} 6.0.3:
+
+* xref:general-bug-fixes[General bug fixes]
+* xref:known-issues[Known issues]
+
+[[general-bug-fixes]]
+== General bug fixes
+
+{productname} 6.0.3 provides fixes for the following bugs:
+
+=== When multiple cells were selected, removing values in the cell properties dialog did not work
```

Comment:
Should we just squash these 2 into one entry (cell and row properties)? They are effectively the same thing and looking back I'm thinking we should have done that with the changelog as well.

Reply:
That was my inclination before starting, but I was following the _each ChangeLog entry is a separate change, requiring separate documentation_ modus operandi.

I presumed, further, there was at least some chance folks in the wild had encountered the bug one way but not the other, which is further (albeit very small) justification for writing the two Changelog entries up separately.

Reply:
All this said, smooshing this into one entry is easily done.

Reply:
Yeah I'm not sure what to do in all honesty. Happy to leave it as is or to squash it, that is to say whatever you think is best 😄

Reply:
On the _it’s not sufficiently broken to require fixing_ principal, and the further _this follows the extant form and following said form does have utility_ principal (in this case the one-to-one relationship between Changelog entries and Release Note entries), I’m going the lazy route and leaving this be.